terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -1146,8 +1146,9 @@ module "slemicro51-sshminion" {
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
 
-  additional_packages = [ "venv-salt-minion" ]
-  install_salt_bundle = true
+//  WORKAROUND until https://bugzilla.suse.com/show_bug.cgi?id=1208045 gets fixed  
+//  additional_packages = [ "venv-salt-minion" ]
+//  install_salt_bundle = true
```

Comment:
I wonder if we should change sumaform, so `install_salt_bundle` has as default a true value. This will align our sumaform deployments with the official doc for 4.3 and 5.0

About `additional_packages = [ "venv-salt-minion" ]` I wonder if we really need this here, in theory, in the cloud-init phase we are also forcing to install this package for almost all the Operating Systems (very few keep using the salt-minion package)

Reply:
`install_salt_bundle` having by default a true value -> absolutely

and later on we could even remove these 2 variable and completely forget about regular salt-minion